addon/index.js

```diff
@@ -188,6 +280,26 @@ export class EmberChangeset extends BufferedChangeset {
 
     return this;
   }
+
+  get(key) {
+    safeGet(this[VIRTUAL_MIRROR], key); //consume tag for native tracked dependencies
+    return super.get(...arguments);
+  }
```

Comment:
Does this handle the `obj.key` case as well?  Or do we have the ability to handle this case?

Reply:
I think yes? Because of the Proxy traps all access 🤔